frontend/src/components/GamesIndex/GamesIndex.jsx

```diff
@@ -1,25 +1,70 @@
-import { useEffect } from "react";
+import { useEffect, useState } from "react";
 import "./GamesIndex.css";
 import { useDispatch, useSelector } from "react-redux";
 import { fetchGames } from "../../store/games";
 import GameIndexItem from "../GameIndexItem/GameIndexItem";
+import jwtFetch from "../../store/jwt";
 
 function GamesIndex() {
-  const dispatch = useDispatch();
-  const games = Object.values(useSelector((state) => state.games));
+  // const dispatch = useDispatch();
+  // const games = Object.values(useSelector((state) => state.games));
```

Comment:
Feel free to remove this code as well. Now that you have the API hooked we don't need it.